### DISCUSSION TITLE
使用curl调用StreamLoad导入csv数据，表中某一列是csv文件中的多列拼接，类似column_u=a_b_c这种格式。请问在header中该怎么写？

### DISCUSSION THREAD

**Discussion by flyzyh** [apache/doris#35167] [A - General / Q&A]
找了各种文档，也没找到类似的例子。官网上只给了列之间做四则运算的写法，对字符串拼接也没举例。
这种情况下，应该使用column_u=CONCAT(a,"_",b,"_",c)，还是简单的加减就行了？
shell调用curl里面各种转义字符写的恶心，试了好几种都不成功。

**Comment by flyzyh** [comment]
在shell里拼CONCAT真的是太恶心了，逗号要转义，引号要转义。而且转义出来的东西也不一定靠谱。
要不下次搞个json的参数吧

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题

**Reply by flyzyh** [reply]
selectdb我也提问了
